fiat-web/src/main/java/com/netflix/spinnaker/fiat/Main.java

```diff
@@ -44,6 +45,7 @@
       // https://github.com/spring-projects/spring-ldap/issues/473
       LdapHealthContributorAutoConfiguration.class
     })
+@SpringBootApplication
```

Comment:
Redundant but also needed for the tck. `SpringPackageVersionResolver` doesn't work without it: https://github.com/spinnaker/kork/blob/master/kork-core/src/main/java/com/netflix/spinnaker/kork/version/SpringPackageVersionResolver.java#L44